modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -122,7 +149,7 @@ The index name must be unique among all indexes and constraints.
 ====
 
 The `CREATE VECTOR INDEX` command take an `OPTIONS` clause. This has two parts, the `indexProvider` and `indexConfig`.
-The provider can only have the default value, `'vector-1.0'`.
+As of Neo4j 5.18, there are two available index providers, `vector-2.0` (default, preferred) and `vector-1.0` (compatibility).
```

Comment:
I'd be inclined to just go with "(default)" here (it kind of assumes that it is preferred), and also remove "(compatibility)" which in it self isnt really telling the user anything? Or am I missing something?

```suggestion
As of Neo4j 5.18, there are two available index providers, `vector-2.0` (default) and `vector-1.0`.
```